R/FindRecalSeries.R

```diff
@@ -20,23 +20,28 @@
 #' @param df An output from RecalList, containing recalibrant CH2 series.
 #' @return A dataframe of 10 best-scoring series.
 
+filter_input <- function(df, abundance_score_threshold, peak_distance_threshold) {
```

Comment:
```suggestion
filter_recal_series <- function(df, abundance_score_threshold, peak_distance_threshold) {
```

Reply:
resolved in f65564ba11ac8bf159e5aea625b8cbd5f9d17581